app/controllers/errors_controller.rb

```diff
@@ -20,6 +21,16 @@ def incorrect_passcode
     render status: 401
   end
 
+  private
```

Comment:
I don't think the login_error method should be private. That is causing the failing specs in Travis Build. Also, there are some Rubocop linting errors in the code. Please have a look at the Travis build for more info.

Reply:
yeah. I just pushed the fix.